### PR TITLE
Format files using flake8 and black

### PR DESCRIPTION
**Pull request by ajlee21**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
           # Enable linters
           black: true
           flake8: true
+          flake8_args: "--ignore=E501"
```
Comment (cgreene) [inline .github/workflows/lint.yaml:29]: I'd recommend using setup.cfg for it so that it will match your local linting environment: https://github.com/greenelab/ponyo/blob/master/setup.cfg

**Comment by ajlee21** [comment]
@cgreene Looks like it passed with 3 errors, see `helper_vae.py` in https://github.com/greenelab/ponyo/pull/6/files

Not sure if we want to add this to the cfg file?
`Line break before binary operator (W503)`

File: ponyo/cca_core.py [hunk 2]
```diff
@@ -159,7 +159,7 @@ def compute_ccas(sigma_xx, sigma_xy, sigma_yx, sigma_yy, verbose=True):
     try:
         ux, sx, vx = np.linalg.svd(arr_x_stable)
         uy, sy, vy = np.linalg.svd(arr_y_stable)
-    except:
+    except BaseException:
```
Comment (cgreene) [inline ponyo/cca_core.py:162]: Is there anything more specific that you expect to get here? Also, if this isn't your code, you could add ignoring it to your flake config and revert this change if you want.

**Comment by cgreene** [comment]
I think you should ignore 503 and add it to the config. It says:
> This rule goes against the PEP 8 recommended style, which was changed on April 16th, 2016 in this commit. The tool will soon be updated to recommend the opposite: line breaks should occur before the binary operator because it keeps all operators aligned.

It looks like the new behavior is 504: https://www.flake8rules.com/rules/W504.html

**Comment by cgreene** [comment]
@dongbohu - can you take a look at the builds on this that were run through CI and see if the linter comments would be helpful during a review? I'm considering making using this linter via github actions the default recommendation for the lab's python code.

File: ponyo/cca_core.py (hunk 2 above)
Comment (ajlee21) [reply ponyo/cca_core.py]: Good point. This script was imported and so I will revert the changes and add the file to the setup.cfg

**Comment by dongbohu** [comment]
> @dongbohu - can you take a look at the builds on this that were run through CI and see if the linter comments would be helpful during a review? I'm considering making using this linter via github actions the default recommendation for the lab's python code.

@cgreene The linter outputs look reasonable to me. I think it is a good starting point. We can (probably will) tweak it further later.
